flask_dance/contrib/slack.py

```diff
@@ -61,14 +64,16 @@ def make_slack_blueprint(
     :returns: A :ref:`blueprint <flask:blueprints>` to attach to your Flask app.
     """
     scope = scope or ["identify", "chat:write:bot"]
+    subdomain = subdomain or ""
+    if subdomain is not "": subdomain += "."
     slack_bp = SlackBlueprint(
         "slack",
         __name__,
         client_id=client_id,
         client_secret=client_secret,
         scope=scope,
         base_url="https://slack.com/api/",
-        authorization_url="https://slack.com/oauth/authorize",
+        authorization_url="https://%sslack.com/oauth/authorize" % subdomain,
```

Comment:
I prefer the `"".format()` function rather than `printf`-style formatting. I suggest this, instead:
```suggestion
        authorization_url="https://{hostname}/oauth/authorize".format(hostname=hostname)
```

Could you please do the same for `token_url`, and manually test that it works properly?